ledger/participant-state-index/src/main/scala/com/daml/ledger/participant/state/index/impl/inmemory/InMemoryUserManagementStore.scala

```diff
@@ -13,7 +13,10 @@ import com.daml.logging.LoggingContext
 import scala.collection.mutable
 import scala.concurrent.Future
 
-class InMemoryUserManagementStore(createAdmin: Boolean = true) extends UserManagementStore {
+class InMemoryUserManagementStore(
```

Comment:
As InMemoryUserManagementStore is now applied in all in memory storage scenarios, it quickly uncovered wholes in conformance: missing support for user right maximum limitation. This is a band-aid to fix that (and make conformance tests pass)